### DISCUSSION TITLE
Sending Bulk eMail from Hosting Service

### DISCUSSION THREAD

**Discussion by acedrummond** [PHPMailer/PHPMailer#2825] [General]
I have a very basic issue which may have come up before however I have not found reference to same in my research.

I am using phpmailer to send an email to each of our members now numbering about 300.

The way I do this in .php is by adding sleep(6) between emails to work around host service restrictions .

Is there any way to send i.e. multiple bcc:'s so that the process is not long running that won't trip the hosting limits?

Is there a limit on how many bcc's phpmailer will accept? or another method?

**Comment by decomplexity** [comment]
You could simply change your email hosting supplier! The 'bulk senders' such as Sendgrid and Mailchimp are fine for sending out bulk marketing emails, but - since the bulk senders' IP addresses may be marked as spam by the recipient email servers - if you need reliable delivery you need to use one of the services that also send out huge amounts of non-bulk (e.g. single addressee) email.
And for this you probably need to pay; both MSFT O365 (who have an addressee limit of 500) and GMail (who also have - or had - 500) work fine with PHPMailer.
